Formula/p/plakar.rb

```diff
@@ -0,0 +1,21 @@
+class Plakar < Formula
+  desc "Effortless Open-Source Backup"
+  homepage "https://plakar.io/"
+  url "https://github.com/PlakarKorp/plakar.git", branch: "main", revision: "main"
+  version "1.0.0"
+  license "ISC"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
This shouldn't be needed for modern software, please check how other go software builds